src/sb/app/project/data.cljc

```diff
@@ -128,4 +128,12 @@
                                                            #{:role/project-admin})]
     (validate/assert project :project/as-map)
     (db/transact! [membership])
-    {:entity/id (:entity/id project)}))
\ No newline at end of file
+    {:entity/id (:entity/id project)}))
+
+(q/defx delete!
+  "Mutation fn. Marks project as deleted by given project-id."
+  {:endpoint {:post "/p/:project-id/delete"}}
```

Comment:
Could we verify if this metadata is still necessary? I wonder if it was left over in the org delete endpoint (my fault), but no longer required or used.

Reply:
Just confirmed that it is not necessary.